mautic/mautic
# Fix this in campaign action

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | NA
| Related developer documentation PR URL | NA
| Issue(s) addressed                     | Fixes https://github.com/mautic/mautic/issues/8963




#### Description:

After M3 migration this campaign is in wrong format

![image](https://user-images.githubusercontent.com/462477/99642449-d20e5300-2a4b-11eb-80a4-2525c3b6db05.png)




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Check campaign change action in campaigns If this is in properly format 

![image](https://user-images.githubusercontent.com/462477/99642529-e7837d00-2a4b-11eb-8685-af0821ba58fa.png)
